Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM docker.io/openshift/origin-release:golang-1.10
+
+#install clang and gcc-c++ required for jsonnet to build
+RUN yum install -y clang \
+    gcc-c++
+
+#get required go packages for building k8s objects
+RUN go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb \
+    github.com/brancz/gojsontoyaml
+
+#building jsonnet
+RUN git clone https://github.com/google/jsonnet && \
+    git --git-dir=jsonnet/.git checkout v0.10.0 && \
+    make -C jsonnet CC=clang CXX=clang++ && \
+    cp jsonnet/jsonnet /usr/bin
+
+#install kubectl package in container
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && \
```

Comment:
I'd suggest just getting kubectl from yum if possible.

Reply:
There are two ways provided by k8s
- https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-using-native-package-management
- https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-using-curl

I found the curl cleaner then configuring the repo. Let me know your thoughts.

Reply:
My concern is what this will look like during downstream builds. I don't think it's permitted to reach out via curl to install stuff that will be in the final package.

I checked centos, and it's quite old:
```
$ docker run -it --rm centos:7 yum info -q kubernetes-client
Available Packages
Name        : kubernetes-client
Arch        : x86_64
Version     : 1.5.2
Release     : 0.7.git269f928.el7
Size        : 14 M
Repo        : extras/7/x86_64
Summary     : Kubernetes client tools
URL         : k8s.io/kubernetes
License     : ASL 2.0
Description : Kubernetes client tools like kubectl
```

It's probably fine to leave these lines as-is, and we'll figure out what to do downstream when the time comes.

Reply:
Ack! Will keep a note of it.